Task_DEV_6/Task_DEV_6/CustomStack.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+namespace Task_DEV_6
+{
+    class CustomStack
+    {
+        private const int LENGTH = 10;
+        private const string EXCEPTION_MESSAGE = "There are no elements on the stack.";
+
+        private int[] elements;
+        private int count;
+
+        public CustomStack()
+        {
+            elements = new int[LENGTH];
+        }
+
+        public CustomStack(int capacity)
```

Comment:
Just advice: if you started to use variable with name `length` above it is better to use such naming here too instead of `capacity`. Because the both of them have the same sense.

Reply:
Done.